### ISSUE TITLE
Dual axis display: hyperparams/metrics vs. other metrics

### ISSUE THREAD

**Issue by michaelitvin** [allegroai/clearml#151]
I would like to display hyperparameters (such as learning rate or batch size) along with some metrics (loss, accuracy etc.), on both train and test sets. Y-scaling should be separate but both plots share the same X.

**Comment by bmartinn** [comment]
@michaelitvin 

This is interesting , I can think of a few options:

1) Use matplot  lib, , it will be automatically caught and presented in the UI as well (no need to worry about running headless by *trains-agent*, it is taken care of) (see [example](https://github.com/allegroai/trains/blob/577c0a3c4196d9ed3630a081e6e16bde42a13de1/examples/frameworks/matplotlib/matplotlib_example.py#L18))

2) Use plotly to generate the plot and then use (see [example](https://github.com/allegroai/trains/blob/577c0a3c4196d9ed3630a081e6e16bde42a13de1/examples/reporting/plotly_reporting.py#L18))
```python
Logger.current_logger().report_plotly(title="accuracy", series="accuracy", iteration=0, figure=fig)
```
3) specify Y-axis title (but unfortunately this is a single title for both axis ) ([example](https://github.com/allegroai/trains/blob/577c0a3c4196d9ed3630a081e6e16bde42a13de1/examples/reporting/scatter_hist_confusion_mat_reporting.py#L62))
```python
Logger.current_logger().logger.report_scatter2d(
        "example_scatter",
        "series_xy",
        iteration=iteration,
        scatter=scatter2d,
        xaxis="title x",
        yaxis="title y",
    )
```

**Comment by michaelitvin** [comment]
Right, but it would be nice to have the ability to do this even if I didn't
think of it in advance. For example, after running an experiment I see that
the variance of the loss decreased at certain time points. I used LR
scheduling, and I want to check if the LR/loss patterns match.

**Comment by bmartinn** [comment]
@michaelitvin how would adding a title for the y-axis help you?
(BTW, you can always disable one of the series by clicking on it in the legend)

**Comment by michaelitvin** [comment]
Adding just a title won't help, what I'm saying is to draw two Y axes, one
on the left side and one on the right. Each has a different scale.

Similar to this:
https://images.squarespace-cdn.com/content/v1/5096baa8e4b09e89382757ce/1353580395848-V5OSX29KKOV6HHLRFF5J/ke17ZwdGBToddI8pDm48kA2lIr9vJgqE_zqHbWQivrpZw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZUJFbgE-7XRK3dMEBRBhUpznHqivAuu2Iyp1cY-CwGmDP5zPBqHHcAdrwS9FyeV6yFpz9Omu9CTOMIEGYVKIx38/image-asset.png?format=1000w

Then some UI would be needed in order to add the desired series to that
graph (e.g. LR vs. accuracy and MSE).

**Comment by michaelitvin** [comment]
Adding just a title won't help, what I'm saying is to draw two Y axes, one on the left side and one on the right. Each has a different scale.

Similar to this:
![image](https://user-images.githubusercontent.com/5813969/85573498-9f982f80-b63e-11ea-88f8-99594814a279.png)

Then some UI would be needed in order to add the desired series to that graph (e.g. LR vs. accuracy and MSE).

**Comment by bmartinn** [comment]
@michaelitvin I see what you mean now.
May I suggest posting this suggestion under #81 , this is where we try to collect all the UI suggestions from users, and slowly but surely add them into Trains :)

**Comment by michaelitvin** [comment]
Great, I see you already did. Thanks!

**Comment by allegroai-git** [comment]
Closing this as this is posted under #81
